.gitignore

```diff
@@ -43,3 +43,10 @@ build/config/buildinfo.properties
 
 www-built
 www-ghdeploy
+/node_modules/
+/tmp/
+/scripts/kiwix-*.pem
+/scripts/set_secret_environment_variables.sh
```

Comment:
I just realised, you protect the secret file using .gitignore - likewise, please ignore my previous comment :)